### ISSUE TITLE
Incompatible dependency version set for seemly

### ISSUE THREAD

**Issue by bkis** [tusen-ai/naive-ui#7232]
### Describe the bug

The new internal `getWCAGContrast` function [relies on `seemly.rgba` to support hsla string input](https://github.com/blackawn/naive-ui/blob/9adad0d6434429e54901f97dc5ad673820c5ce35/src/color-picker/src/utils.ts#L64). This was [introduced in seemly v3.10](https://github.com/07akioni/seemly/blob/a95246de72117be072407de49df4623f5d805530/src/color/index.ts#L136), but Naive-UI's dependency requirement for seemly [is set to `^0.3.8`](https://github.com/tusen-ai/naive-ui/blob/acb01b6c14590079e9d20a455ed19051794f935d/package.json#L86), allowing **incompatible seemly versions** `0.3.8` and `0.3.9`, which will lead to errors:

```
Uncaught (in promise) Error: [seemly/rgba]: Invalid color value hsla(140, 34%, 33%, 1).
    rgba index.js:120
    getWCAGContrast utils.mjs:28
    setup ColorPickerTrigger.mjs:57
    renderComponentRoot runtime-core.esm-bundler.js:6592
    componentUpdateFn runtime-core.esm-bundler.js:5392
    run reactivity.esm-bundler.js:237
    setupRenderEffect runtime-core.esm-bundler.js:5527
    mountComponent runtime-core.esm-bundler.js:5301
    processComponent runtime-core.esm-bundler.js:5253
    patch runtime-core.esm-bundler.js:4769
    componentUpdateFn runtime-core.esm-bundler.js:5399
    ...
```

### Steps to reproduce

1. Install Naive-UI `2.43.0` or above
2. Force to use seemly `0.3.8` or `0.3.9` by installing it
3. Try to use `n-color-picker`

### Link to minimal reproduction

https://codesandbox.io/p/sandbox/zealous-scott-fqkltd

### System Info

```Shell
System:
    OS: Linux 6.16 Pop!_OS 22.04 LTS
    CPU: (8) x64 AMD Ryzen 7 PRO 3700U w/ Radeon Vega Mobile Gfx
    Memory: 2.85 GB / 13.54 GB
    Container: Yes
    Shell: 5.1.16 - /bin/bash
  Binaries:
    Node: 22.19.0 - /usr/bin/node
    Yarn: 1.22.19 - /usr/bin/yarn
    npm: 10.9.3 - /usr/bin/npm
    pnpm: 10.0.0 - /usr/bin/pnpm
  Browsers:
    Chromium: 112.0.5615.165
```

### Used Package Manager

npm

### Validations

- [x] Read the [Contributing Guidelines](https://github.com/tusen-ai/naive-ui/blob/main/CONTRIBUTING.md).
- [x] Read the [docs](https://www.naiveui.com/en-US/).
- [x] Check that there isn't [already an issue](https://github.com/tusen-ai/naive-ui/issues) that reports the same bug to avoid creating a duplicate.
- [x] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/tusen-ai/naive-ui/discussions).
- [x] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by sfw5201314** [comment]
啥时候更新啊这个bug

**Comment by jahnli** [comment]
等下次发版吧，可以先 patch 下
